app/assets/javascripts/util/restInterface.js

```diff
@@ -157,7 +157,7 @@ export default {
     basePath += `?${queryString.stringify(params)}`;
 
     return this.handleRequest(
-      fetch(path, {
+      fetch(basePath, {
```

Comment:
I need to clean up this file. Well, mainly I need to delete it. I'll get to this later. This is really only being used for grabbing the auth token.